extensions/wikia/PortableInfobox/querypage/AllinfoboxesQueryPage.php

```diff
@@ -104,9 +104,14 @@ public function reallyDoQuery( $limit = false, $offset = false ) {
 			} );
 
 		return array_filter( $result, function ( $tmpl ) {
-			$data = PortableInfoboxDataService::newFromPageID( $tmpl[ 'pageid' ] )->getData();
+			$title = Title::newFromID( $tmpl[ 'pageid' ] );
+			if ( $title && $title->exists() && !$title->isSubpage() ) {
+				$data = PortableInfoboxDataService::newFromTitle( $title )->getData();
```

Comment:
This question should be directed to Jakub. However this approach shields us from changes in names of subpages (especially translation connected). Is there any major opposing point for that?